common/djangoapps/student/views.py

```diff
@@ -567,9 +567,17 @@ def dashboard(request):
 
     message = ""
     if not user.is_active:
+        email_provider = user.email.split("@")[1].split(".")[0].lower()
+        if email_provider=='gmail':
+            spam_name = pgettext("gmail", "Spam")
```

Comment:
Include a `# Translators: ` note like:

```
# Translators: Use the Gmail's translation for the language and include the English original name between parentheses just in case the learner is using the English Gmail interface which is very common.
spam_name = pgettext("gmail", "Spam")
```